src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/SQLSetupStrategy.cs

```diff
@@ -277,7 +278,9 @@ public PlatformSpecificTestContext()
 
         public void Dispose()
         {
-            Fixture.Dispose();
+            akvFixture?.Dispose();
```

Comment:
If any of these wrap system resources (cert store?) it might be worth putting a try-finally around each invoke so that even if one of them fail the others will still be tried. It doesn't matter for managed resources since they'll be freed when the process dies but for things like pinvokes that can be system-wide.